packages/@aws-cdk/aws-ec2-alpha/test/integ.vpc-v2-alpha.js.snapshot/aws-cdk-vpcv2-alpha.template.json

```diff
@@ -9,7 +9,7 @@
     "InstanceTenancy": "default"
    }
   },
-  "VPCintegtest1SecondaryAddress256BAC1D3": {
+  "VPCintegtest1SecondaryAddress2B60D56E9": {
```

Comment:
do you know why these logical ids got changed although you did not change anything in the testing stack

Reply:
we need to be careful about the logical ids changes, as it means that the old resources will be deleted, and create new resources, and this can be a breaking change for the customers.

Reply:
I believe it is related to the change in `packages/@aws-cdk/aws-ec2-alpha/lib/vpc-v2.ts` when you replaced the `CfnVPCCidrBlock` resource with the new L2 resource. Could you please check if this is a breaking change (I mean if it will cause any outage for the customers or not during the update). If yes, I think it is ok since this module is an alpha module, but we need to highlight the change as a breaking change.

Reply:
I verified that this is a breaking change. The stacks that got already deployed using the old construct will fail to be updated after this change and will get this error `CIDR range conflicts with x.xx.xx.xx/xx with association ID vpc-cidr-assoc-ABCD` .. we should highlight this issue as a breaking change.